src/set_theory/game/pgame.lean

```diff
@@ -504,14 +509,14 @@ end
 
 /-- Define the equivalence relation on pre-games. Two pre-games
   `x`, `y` are equivalent if `x ≤ y` and `y ≤ x`. -/
-def equiv (x y : pgame) : Prop := x ≤ y ∧ y ≤ x
+def equiv : pgame → pgame → Prop := @antisymm_rel pgame (≤)
 
 local infix ` ≈ ` := pgame.equiv
```

Comment:
What happens if we remove this entirely and put the
```
instance pgame.setoid : setoid pgame :=
antisymm_rel.setoid pgame (≤)
```
here? That should provide the same notation through `has_equiv.equiv`.

Reply:
We shouldn't make that setoid an instance, because we'll soon have another one (pre-games up to relabelling).

Reply:
Well, it seems like we decided against that new instance. I do still believe we need to think about how we're going to deal with all of our equivalence notions on pre-games: identity, relabellings, and equivalence. But that can wait for later.

Even then, I'm not 100% sure that we want to get rid of the def. That seems like it might break dot notation. Would it?

Reply:
Yes, I think it will break notations unless you moves the lemmas under `has_equiv.equiv` namespace, but we don't seem to have general typeclasses/predicates for an equivalence to be compatible with `+`, `*`, etc. So I think keeping `pgame.equiv` is useful and now I'm not so sure whether I should remove `surreal.equiv` in #14585.